halo2-lib-js/src/halo2lib/functions.ts

```diff
@@ -13,19 +13,23 @@ export class Halo2Lib {
     constructor(halo2wasm: Halo2Wasm, circuit: Halo2LibWasm, options?: { firstPass?: boolean }) {
         this. _halo2lib = circuit;
         this._firstPass = options?.firstPass ?? false;
-        this._MAX_BITS = this.getPaddedNumBits("253");
+        this._MAX_BITS = this.getMaxPaddedNumBits();
         this._halo2wasm = halo2wasm;
     }
 
     private Cell(a: number) {
         return new CircuitValue(this. _halo2lib, { cell: a });
     }
 
-    private getPaddedNumBits(numBits: string) {
-        let numBitsNum = Number(numBits);
+    private getMaxPaddedNumBits() {
         const lookupBits = this. _halo2lib.lookup_bits();
-        let paddedC = Math.floor(numBitsNum / lookupBits) * lookupBits - 1;
-        return paddedC.toString();
+        let maxPaddedNumBits = Math.floor(253 / lookupBits) * lookupBits - 1;
+        return maxPaddedNumBits.toString()
+    }
+
+    private getPaddedNumBits(numBits: string) {
```

Comment:
This function is now confusingly named, since we aren't adding padding anymore.  Maybe something like `getValidatedNumBits`?